MixItUp.WPF/Controls/Services/OvrStreamServiceControl.xaml.cs

```diff
@@ -0,0 +1,79 @@
+﻿using MixItUp.Base;
+using MixItUp.WPF.Util;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Navigation;
+
+namespace MixItUp.WPF.Controls.Services
+{
+    /// <summary>
+    /// Interaction logic for OvrStreamServiceControl.xaml
+    /// </summary>
+    public partial class OvrStreamServiceControl : ServicesControlBase
+    {
+        public OvrStreamServiceControl()
+        {
+            InitializeComponent();
+        }
+
+        protected override async Task OnLoaded()
+        {
+            this.SetHeaderText("OvrStream");
+
+            if (!string.IsNullOrEmpty(ChannelSession.Settings.OvrStreamServerIP))
+            {
+                this.OvrStreamIPAddressTextBox.Text = ChannelSession.Settings.OvrStreamServerIP;
+
+                this.OvrStreamEnableConnectionButton.Visibility = Visibility.Collapsed;
+                this.OvrStreamDisableConnectionButton.Visibility = Visibility.Visible;
+
+                this.SetCompletedIcon(visible: true);
+            }
+            else
+            {
+                this.OvrStreamIPAddressTextBox.Text = ChannelSession.DefaultOvrStreamConnection;
+            }
+
+            await base.OnLoaded();
+        }
+
+        private void Hyperlink_RequestNavigate(object sender, RequestNavigateEventArgs e)
+        {
+            // Process.Start("OBS");
```

Comment:
I left this in here in case we wanted to expose a link to their website, it isn't hooked up to anything, I could remove it.